src/main/java/frc/robot/commands/Shoot.java

```diff
@@ -29,26 +43,32 @@ public void initialize() {
   @Override
   public void execute() {
     subShooter.setShootingVelocities(
-        prefShooter.leftShooterVelocity.getValue(),
-        prefShooter.leftShooterFeedForward.getValue(),
-        prefShooter.rightShooterVelocity.getValue(),
-        prefShooter.rightShooterFeedForward.getValue());
+        this.leftShooterVelocity.getValue(),
+        this.leftShooterFeedForward.getValue(),
+        this.rightShooterVelocity.getValue(),
+        this.rightShooterFeedForward.getValue());
 
     // Set LEDs when shooters are up to speed
-    if (subShooter.isLeftShooterAtVelocity(prefShooter.leftShooterVelocity.getValue(),
+    if (subShooter.isLeftShooterAtVelocity(leftShooterVelocity.getValue(),
         prefShooter.shooterUpToSpeedTolerance.getValue())
-        && subShooter.isLeftShooterAtVelocity(prefShooter.leftShooterVelocity.getValue(),
+        && subShooter.isRightShooterAtVelocity(rightShooterVelocity.getValue(),
```

Comment:
This currently only checks if our flywheels are up to speed. The name of this pr suggests that we would also like to check if our Pitch angle (and turret angle?) are also within tolerance. Either change the name of the pr or add a check for those angles.